docs/framework/wcf/feature-details/comparing-aspnet-web-services-to-wcf-based-on-development.md

```diff
@@ -182,15 +182,15 @@ public class LineItem
   
 -   As a consequence of being able to serialize the non-public members of types into XML, the <xref:System.Runtime.Serialization.DataContractSerializer> has fewer restrictions on the variety of .NET types that it can serialize into XML. In particular, it can translate into XML types like <xref:System.Collections.Hashtable> that implement the <xref:System.Collections.IDictionary> interface. The <xref:System.Runtime.Serialization.DataContractSerializer> is much more likely to be able to serialize the instances of any pre-existing .NET type into XML without having to either modify the definition of the type or develop a wrapper for it.  
   
--   Another consequence of the <xref:System.Runtime.Serialization.DataContractSerializer> being able to access the non-public members of a type is that it requires full trust, whereas the <xref:System.Xml.Serialization.XmlSerializer> does not. The Full Trust code access permission give complete access to all resources on a machine that can be access using the credentials under which the code is executing. This options should be used with care as fully trusted code accesses all resources on your machine.  
+-   Another consequence of the <xref:System.Runtime.Serialization.DataContractSerializer> being able to access the non-public members of a type is that it requires full trust, whereas the <xref:System.Xml.Serialization.XmlSerializer> does not. The Full Trust code access permission gives complete access to all resources on a machine that can be accessed using the credentials under which the code is executing. This options should be used with care as fully trusted code accesses all resources on your machine.  
```

Comment:
"This options" should be "This option"

Reply:
Haven't noticed this one, thanks. Addressed it in the second commit.